web/docs/map-data-to-ocsf/README.md

```diff
@@ -0,0 +1,846 @@
+# Map Data to OCSF
+
+In this tutorial you'll learn how to map events to [Open Cybersecurity Schema
+Framework (OCSF)](https://schema.ocsf.io). We walk you through an example of
+events from a network monitor and show how you can use Tenzir pipelines to
+easily transform them so that they become OCSF-compliant events.
+
+![OCSF Pipeline](ocsf-pipeline.svg)
+
+The diagram above illustrates the data lifecycle and the OCSF mapping takes
+place: you collect data from various data sources, each of which has a different
+shape, and then convert them to a standardized representation. The primary
+benefit is that **normalization decouples data acquisition from downstream
+analytics**, allowing the processes to scale indepedently.
+
+## OCSF Primer
+
+The OCSF is a vendor-agnostic event schema (aka. "taxonomy") that defines
+structure and semantics for security events. Here are some key terms you need to
+know to map events:
+
+- **Attribute**: a unique identifier for a specific type, e.g., `parent_folder` of type `String` or `observables` of type `Observable Array`.
+- **Event Class**: the description of an event defined in terms of attributes,
+  e.g., `HTTP Activity` and `Detection Finding`.
+- **Category**: a group of event classes, e.g., `System Activity` or `Findings`.
+
+The diagram below illustrates how subsets of attributes form an event class:
+
+![OCSF Event Classes](ocsf-event-classes.svg)
+
+The **Base Event Class** is a special event class that's part of every event
+class. Think of it as a mixin of attributes:
```

Comment:
Mixin is a very technical term. Can you find something easier to understand?

Reply:
Slightly clarified it.